### PR TITLE
added Clive instructions to readme

### PR DESCRIPTION
**Pull request by regatta-sab**
- Added Clive instructions to readme

### CODE REVIEW

**Comment by BlaineTrain1995** [comment]
Greetings Sam,

I have noticed a lack of care in your work, shoddy really. I do not condone this type of reckless behavior. If this continues we will have to write you up for carelessness and lack of professionalism. We recently had held you in high regard, but no longer peasant.

P.s Does anyone want a cup of tea?

Yours sincerely, 
B. Train

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 CSS/JS files needed for Magento
 
 Merge into master to upload to AWS
+
+Run job on Cliiiiiive to deploy files to AWS.
```
Comment (BlaineTrain1995) [inline README.md]: get it fixed lad

Comment (BlaineTrain1995) [inline README.md]: U DONT NEED THAT MANY I'S

**Comment by regatta-sab** [comment]
@BlaineTrain1995  how is that Blaine Train?

**Comment by BlaineTrain1995** [comment]
am very angery

**Pull request merged**